docker/README.md

```diff
@@ -328,6 +339,10 @@ Start the default services, the kafka services, and the dictionary service:
 
 ```docker compose --profile quickstart --profile dictionary --profile kafka up -d```
 
+Start the default services, and the file provider service:
+
+```docker compose --profile file-provider up -d```
```

Comment:
would the quickstart profile need to be started beforehand in the case? Or since the file-provider might be independent of it maybe we don't need it?